db/migrate/20170928211111_create_oidc_provider_authorizations.rb

```diff
@@ -1,11 +1,14 @@
+# frozen_string_literal: true
+
 class CreateOIDCProviderAuthorizations < ActiveRecord::Migration[5.1]
   def change
     create_table :oidc_provider_authorizations do |t|
-      t.references :account, foreign_key: {to_table: OIDCProvider.account_class.tableize}, null: false
+      t.references :account, foreign_key: { to_table: OIDCProvider.account_class.tableize }, null: false
       t.string :client_id, null: false
       t.string :nonce
       t.string :code, null: false
       t.text :scopes, null: false
+      t.text :claims
```

Comment:
Completely agreed

Reply:
Oh, sorry, checking my code again and I see that here it will apply for new projects, while the new migration script `db/migrate/20230721112432_add_claims_to_oidc_provider_authorization_if_missing.rb` will add that column for existing projects when the column doesn't exist already, so that it works in both cases.

We could keep it like this, but if you really want it I can remove it.